src/Form/Type/PayzenGatewayConfigurationType.php

```diff
@@ -36,6 +36,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'sylius.form.gateway_configuration.payzen.ctx_mode.choices.production' => Api::MODE_PRODUCTION,
                 ),
             ])
+            ->add('endpoint', ChoiceType::class, [
+                'label'       => 'sylius.form.gateway_configuration.payzen.endpoint.label',
+                'choices'  => array(
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.default' => null,
```

Comment:
Is the `sylius.form.gateway_configuration.payzen.endpoint.choices.default` option implicitly refers to https://secure.payzen.eu/vads-payment/ ?

Reply:
Yes it's the fall back of the API of ekyna/PayzenPayum